SPECS-EXTENDED/rlwrap/rlwrap.spec

```
@@ -0,0 +1,73 @@
+Name:           rlwrap
+Version:        0.46.1
+Release:        1%{?dist}
+Summary:        Wrapper for GNU readline
+URL:            https://github.com/hanslub42/rlwrap
+Source:         https://github.com/hanslub42/rlwrap/releases/download/%{version}/%{name}-%{version}.tar.gz
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  readline-devel
+BuildRequires:  ncurses-devel
+
+%description
+rlwrap is a 'readline wrapper' that uses the GNU readline library to
+allow the editing of keyboard input for any other command. Input
+history is remembered across invocations, separately for each command;
+history completion and search work as in bash and completion word
+lists can be specified on the command line.
+
+
+%prep
+%autosetup -p1
+
+
+%build
+%configure
+%make_build
+
+%install
+%make_install
+
+
+%check
+make check
+
+
+%files
+%license COPYING
+%doc AUTHORS NEWS README.md
+%{_bindir}/rlwrap
+%{_mandir}/*/rlwrap.*
+%{_mandir}/man3/RlwrapFilter.*
+%{_datadir}/rlwrap
+
+
+%changelog
+* Thu Jun 22 2023 Saranya R <rsaranya@microsoft.com> - 0.46.1-1
+- Initial CBL-Mariner import from Fedora (license: MIT).
```

Can you please add what version of Fedora this spec was imported from ?
e.g: F38 has this version
```suggestion
- Initial CBL-Mariner import from Fedora 38 (license: MIT).
```